tests/playwright/pages/wp-admin-page.ts

```diff
@@ -175,10 +175,42 @@ export default class WpAdminPage extends BasePage {
 		await this.page.click( '#submit' );
 	}
 
-	async setLanguage( language: string ) {
```

Comment:
I have added api function
```
export async function setWPLanguage( request: APIRequestContext, language: string ) {
	await _post( request, 'settings', { language } );
}
```
But for some reason it works only locally. If it will work for you, then you can use it

Reply:
@dmaziar I will stick with the old method for now.